helm-charts/splunk-otel-collector/templates/NOTES.txt

```diff
@@ -82,3 +82,7 @@ Splunk OpenTelemetry Collector is installed and configured to send data to Splun
   - Some libraries may be enabled by default. For current status, see: https://github.com/open-telemetry/opentelemetry-operator#controlling-instrumentation-capabilities
   - Splunk provides best-effort support for native OpenTelemetry libraries, and full support for Splunk library distributions. For used libraries, refer to the values.yaml under "operator.instrumentation.spec".
 {{- end }}
+{{- if not (eq (toString ((.Values.agent.config).exporters).sapm) "<nil>") }}
```

Comment:
This should work
```suggestion
{{- if not (eq (toString .Values.agent.config.exporters.sapm) "<nil>") }}
```

Reply:
Also, need to check the same for gateway

Reply:
No, that fails with `make render`:
```
Error: template: splunk-otel-collector/templates/NOTES.txt:85:32: executing "splunk-otel-collector/templates/NOTES.txt" at <.Values.agent.config.exporters.sapm>: nil pointer evaluating interface {}.sapm
```